src/workflows/airtable/activities.ts

```diff
@@ -22,37 +22,44 @@ export interface ICreativesScoresAirtable extends FieldSet {
 }
 
 export interface ISoftrCreativesUser extends FieldSet {
-  Email: string;
+  Email?: string;
   "Token Address": string;
   "Wallet Address": string;
-  "Magic Link": string;
+  "Magic Link"?: string;
   "Start Date"?: string[];
-  Name: string;
+  Name?: string;
   Status: number;
   Level: number;
-  "Profile image"?: string[];
-  "Dashboard - Backend iFrame"?: string[];
+  //"Profile image"?: string[];
```

Comment:
if these columns are not in new bases we better remove them rather than commenting them out

Reply:
This is also resolved